test/env/default.sh

```diff
@@ -436,12 +419,15 @@ fi
 
 if [ "$SUNDIALS_PRECISION" == "double" ]; then
     export SUNDIALS_RAJA=ON
-    export RAJA_ROOT="$(spack location -i raja@0.14.0 % "$compiler")"
+    export RAJA_ROOT="$(spack location -i raja@2022.03.1 ~openmp +cuda %"$compiler")"
     export RAJA_BACKENDS="CUDA"
+    # RAJA does not find camp on its own?
+    export camp_ROOT="$(spack location -i camp@2022.03.2 +cuda %"$compiler")"
```

Comment:
Thanks!  I kept running into this camp issue when compiling SUNDIALS with RAJA support in the past.